src/keymap.cpp

```diff
@@ -139,7 +139,7 @@ static op_desc opdescs[] = {
 	},
 	{
 		OP_SETTAG, "set-tag", "t",
-		_("Select tag"),
+		_("Set tag"),
```

Comment:
> Wouldn't that also reintroduce the typo in src/keymap:37 and the translation files?

It won't, because these changes were made in another commit. But it *will* remove some other edits you did in that same commit. So yeah, splitting it further in the first place would've been better. Luckily with Git it's easy to tweak and rewrite history, and you can always roll back—see `git reflog`.

Your new commit is OK. I'll rewrite history (with git rebase -i) and squash it into the old one to make it look like you never made that particular change. I prefer to keep history clean.